### PR TITLE
$ref for mef-persons

### PR DESCRIPTION
**Pull request by rerowep**
For testing the setup has to be run!
See links from Documents to Authors and Document links from Author

### CODE REVIEW

File: rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html [hunk 1]
```diff
@@ -51,6 +51,22 @@ <h1 class="mb-0">{{ record | person_label(current_i18n.language) }}</h1>
     </section>
   </article>
 </article>
+<!-- Documents -->
+{%- if record.documents %}
+<section class="py-4 border-top">
```
Comment (iGormilhit) [inline rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html]: py-4 is too much, pt-4 is enough, I think.

File: rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html [hunk 2]
```diff
@@ -51,6 +51,22 @@ <h1 class="mb-0">{{ record | person_label(current_i18n.language) }}</h1>
...
+  <header class="mb-4">
```
Comment (iGormilhit) [inline rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html]: I think it's not necessary to add mb-4. The h3 below has already a margin-bottom.

File: rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json [hunk 3]
```diff
@@ -53,6 +53,23 @@
             },
             "qualifier": {
               "type": "keyword"
+            },
+            "mefidentifiers": {
```
Comment (iGormilhit) [inline rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json]: I'm not sure for this too. After the $ref resolution, what do we have in the document JSON? These identifiers?

File: data/organisations-libraries-locations.json [hunk 4]
```diff
@@ -0,0 +1,512 @@
+[
```
[the remark this answers is not in the document]
Comment (rerowep) [reply data/organisations-libraries-locations.json]: Ok

File: rero_ils/config.py [hunk 5]
```diff
@@ -682,12 +682,13 @@ def _(x):
...
-        template='rero_ils/detailed_view_persons.html',
-        record_class='rero_ils.modules.mef_persons.api:MefPerson',
-    )
+    # 'pers': dict(
+    #     pid_type='pers',
+    #     route='/persons/<pid_value>',
+    #     template='rero_ils/detailed_view_persons.html',
+    #     view_imp='rero_ils.modules.mef_persons.views.persons_view_method',
+    #     record_class='rero_ils.modules.mef_persons.api:MefPerson',
+    # )
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/config.py]: ok

File: rero_ils/modules/documents/jsonresolver_mef_person.py [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+                            date_of_death=date_of_death
+                        )
+                        author['date'] = date
+                    # qualifier ?????
+                    # current_app.logger.info(
+                    #     'Mef resolver: {result} {url}'.format(
+                    #         result=author,
+                    #         url=mef_url
+                    #     )
+                    # )
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/documents/jsonresolver_mef_person.py]: ok

File: data/libraries.json [hunk 7]
```diff
@@ -427,4 +427,4 @@
       }
     ]
   }
-]
+]
```
[the remark this answers is not in the document]
Comment (rerowep) [reply data/libraries.json]: ??

File: data/locations.json [hunk 8]
```diff
@@ -95,4 +95,4 @@
       "$ref": "http://ils.rero.ch/api/libraries/4"
     }
   }
-]
+]
```
[the remark this answers is not in the document]
Comment (rerowep) [reply data/locations.json]: ??

File: data/organisations.json [hunk 9]
```diff
@@ -2,4 +2,4 @@
   "$schema": "http://ils.rero.ch/schema/organisations/organisation-v0.0.1.json",
   "address": "Via Challand 132, 11100 Aosta",
   "name": "R\u00e9seau des biblioth\u00e8ques du Canton d'Aoste"
-}
+}
```
[the remark this answers is not in the document]
Comment (rerowep) [reply data/organisations.json]: ??

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json [hunk 10]
```diff
@@ -307,4 +344,4 @@
       }
     }
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json]: ??

File: rero_ils/modules/item_types/jsonschemas/item_types/item_type-v0.0.1.json [hunk 11]
```diff
@@ -47,4 +47,4 @@
       }
     }
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/item_types/jsonschemas/item_types/item_type-v0.0.1.json]: ??

File: rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html (hunk 1 above)
Comment (rerowep) [reply rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html]: Changed

File: rero_ils/modules/item_types/jsonschemas/form_item_types/item_type-v0.0.1.json [hunk 12]
```diff
@@ -29,4 +29,4 @@
       }
     ]
   }
-]
+]
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/item_types/jsonschemas/form_item_types/item_type-v0.0.1.json]: ??

File: rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html (hunk 2 above)
Comment (rerowep) [reply rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html]: Took this one from Items on document view.

File: rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json (hunk 3 above)
Comment (rerowep) [reply rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json]: the jsonresolver_mef_person.py reduces the $ref to this

File: data/libraries.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (iGormilhit) [reply data/libraries.json]: There's no change, this file shouldn't be commited.

File: rero_ils/modules/item_types/jsonschemas/form_item_types/item_type-v0.0.1.json (hunk 12 above)
[the remark this answers is not in the document]
Comment (iGormilhit) [reply rero_ils/modules/item_types/jsonschemas/form_item_types/item_type-v0.0.1.json]: There's no change, this file shouldn't be commited.

File: rero_ils/modules/item_types/jsonschemas/item_types/item_type-v0.0.1.json (hunk 11 above)
[the remark this answers is not in the document]
Comment (iGormilhit) [reply rero_ils/modules/item_types/jsonschemas/item_types/item_type-v0.0.1.json]: There's no change, this file shouldn't be commited.

**Pull request closed without merging**